files/ru/glossary/abstraction/index.html

```diff
@@ -1,13 +1,52 @@
 ---
 title: Абстракция
 slug: Glossary/Abstraction
+tags:
+  - Абстракция
+  - Программирование
+  - Написание скриптов
+  - Глоссарий
+  - Язык программирования
 translation_of: Glossary/Abstraction
 original_slug: Словарь/Abstraction
 ---
-<p>Абстракция(обобщение) в {{Glossary("computer programming", "программировании")}} это способ дать упрощённое представление о компонентах системы, скрыть их сложность и детали реализации, предоставив подходящий {{Glossary("API", "программный интерфейс")}}.</p>
+<p>Абстракция в {{Glossary("computer programming", "программировании")}} — это способ снизить сложность, обеспечив эффективное проектирование и внедрение, в сложных компьютерных системах. Она скрывает техническую сложность за более простым {{Glossary("API", "API")}}.</p>
 
-<p id="Learn_more"><strong>Дополнительно</strong></p>
+<h2 id="Преимущества_абстракции">Преимущества абстракции</h2>
 
 <ul>
- <li>{{interwiki("wikipedia", "Abstraction (computer science)", "Abstraction")}} on Wikipedia</li>
+ <li>Помогает избежать написания низкоуровневого кода.</li>
+ <li>Позволяет избежать дублирования и упрощает повторное использование кода.</li>
+ <li>Позволяет изменять внутреннюю реализацию не затрагивая пользователей.</li>
+ <li>Помогает повысить безопасность приложения или программы, поскольку пользователям доступны только важные детали.</li>
```

Comment:
Предлагаю переставить местами, чтобы не было в начале "Помогает... Позволяет... Позволяет... Помогает"
```suggestion
 <li>Помогает избежать написания низкоуровневого кода.</li>
 <li>Упрощает повторное использование кода и позволяет избежать дублирования.</li>
 <li>Даёт возможность изменять внутреннюю реализацию программы, не затрагивая пользователей.</li>
 <li>Помогает повысить безопасность приложения или программы, поскольку пользователям доступны только важные детали.</li>
```

Reply:
А ещё, что думаешь, если заменить в последнем тезисе "важные" на "нужные"? Мне кажется, что когда ты пользователям предоставляешь "важные" части программы, безопасностью особо не пахнет. А если "только нужные" - наоборот.